### PR TITLE
config: update config through tikv-ctl

### PR DESCRIPTION
**Pull request by NingLin-P**
### What problem does this PR solve?

This pr do two things:
- register the status server address to pd
- make tikv-ctl can dynamically modify tikv config

Tests 

- Manual test

### CODE REVIEW

File: src/server/status_server.rs [hunk 1]
```diff
@@ -537,6 +549,79 @@ impl StatusServer {
...
+                    Err(e) => error!("failed to register addr to pd"; "error" => ?e),
+                }
+            }
+            // refresh the pd leader
+            if let Err(e) = pd_client.reconnect() {
+                error!("failed to reconnect pd client"; "err" => ?e);
+            }
+        }
+        error!(
+            "failed to unregister addr to pd after {} tries",
```
Comment (rleungx) [inline src/server/status_server.rs:588]:
```suggestion
            "failed to register addr to pd after {} tries",
```

File: src/server/status_server.rs [hunk 2]
```diff
@@ -537,6 +549,79 @@ impl StatusServer {
...
+        let json = {
+            let mut body = std::collections::HashMap::new();
+            body.insert("component".to_owned(), COMPONENT.to_owned());
+            body.insert("addr".to_owned(), status_addr);
+            serde_json::to_string(&body).unwrap()
+        };
+        for _ in 0..COMPONENT_REQUEST_RETRY {
+            for pd_addr in pd_client.get_leader().get_client_urls() {
+                let pd_addr = pd_addr
+                    .trim_start_matches("http://")
```
Comment (rleungx) [inline src/server/status_server.rs:568]: How about using `Url::parse`?

Comment (NingLin-P) [reply src/server/status_server.rs:568]: Is the client urls return from pd always starts with scheme like `http://` or `https://`?

**Pull request merged**